pagefaultgames/pokerogue
# [Ability] Allow Power Construct to transform 10% PC Zygardes

## What are the changes the user will see?

The 10% Power Construct form for Zygarde can now transform into the Complete form, the same as 50% Power Construct Zygardes.  As such, 10% Power Construct Zygardes can now be encountered in the wild, and should also be able to be hatched from eggs as well.

## Why am I making these changes?



The Power Construct ability was still only partially implemented, and this should fully implement it.

## What are the changes from a developer perspective?


All of the form changes attributes have been changed to conditionals to account for the two different forms (50% PC and 10% PC) transforming into Complete form. A new form was also added for Zygarde which is functionally and visually identical to Complete form, with the purpose of it being to be used for the 10% PC Zygardes instead. The `randSeedInt` for Zygarde's `getSpeciesFormIndex` entry was increased by one, to now allow for 10% Power Construct forms to be available for players.

### Screenshots/Videos



https://github.com/user-attachments/assets/3b630a64-d332-4d2e-93bf-5af344d0e7ee

## How to test the changes?




Do a starter form override for Zygarde by using `{ [Species.ZYGARDE]: 3 }` to start with 10% Power Construct Zygarde. Test by getting HP under 50%, and see how it reverts back to 10% PC form when recalled (such as for a trainer).

## Checklist
